recipes/openvdb/config.yml

```diff
@@ -1,3 +1,5 @@
 versions:
   "8.0.1":
     folder: "8.0.1"
+  "8.1.0":
+    folder: "8.1.0"
```

Comment:
The recipes look really really similar... why did you make it a seperate folder?

Reply:
Whoops missed the previous comment

> > I would suggest you using same recipe from 8.0.1 if possible. You can rename that folder to "all" and support any version.
> 
> Since they changed many things in the library cmake options I thought it would be a better idea to have it in a different file instead of filling it with `if-else`

This has typically not been a strong argument... though I must agree... we have not separated versions and made for a lot messy recipes